### PR TITLE
IBM EHLLAPI low code workflow library

### PR DESCRIPTION
**Pull request by smiccile**

### CODE REVIEW

File: snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/hello.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
+
+from flask import Flask, request, jsonify
```
Comment (samratjha96) [inline snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/hello.py:2]: You can remove this whole file. This was just during testing

File: snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/ehllapi_sample/EHLLAPI_Sample.c [hunk 2]
```diff
@@ -0,0 +1,80 @@
+//Include standard library functions
+#include <stdlib.h>
```
Comment (samratjha96) [inline snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/ehllapi_sample/EHLLAPI_Sample.c:2]: Probably also wanna remove this file

File: snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/requirements.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+py3270==0.3.5
```
Comment (samratjha96) [inline snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/requirements.txt:1]: Remove this file as well

File: snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/hello.py (hunk 1 above)
Comment (smiccile) [reply snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/hello.py]: Done

File: snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/ehllapi_sample/EHLLAPI_Sample.c (hunk 2 above)
Comment (smiccile) [reply snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/ehllapi_sample/EHLLAPI_Sample.c:2]: Done

File: snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/requirements.txt (hunk 3 above)
Comment (smiccile) [reply snippets-low-code/snippet-ibm-3270-ehllapi-library/examples/appian python/requirements.txt]: Done

**Pull request merged**